webapp/channels/src/components/admin_console/server_logs/logs.tsx

```diff
@@ -16,17 +16,20 @@ import AdminHeader from 'components/widgets/admin_console/admin_header';
 
 import LogList from './log_list';
 import PlainLogList from './plain_log_list';
+import { Client4 } from 'mattermost-redux/client';
```

Comment:
Could you fix these linting errors? I'm pretty sure that most of these can be fixed automatically if you run `npm run fix`